src/plonemeeting/portal/core/browser/institution.py

```
@@ -56,12 +55,14 @@ class InstitutionSettingsView(DefaultView):
     }
 
     def _update(self):
-        super(InstitutionSettingsView, self)._update()
+        super()._update()
         if "password" in self.w:
-            self.w["password"].value = self.context.password and "********************" or "-"
+            self.w["password"].value = (
+                self.context.password and "********************" or "-"
```

Ruff (0.15.14)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)